pkg/qemu/qemu.go

```diff
@@ -1006,16 +1006,16 @@ func FindVirtiofsd(qemuExe string) (string, error) {
 		Binary      string `json:"binary"`
 	}
 
-	currentUser, err := user.Current()
+	homeDir, err := os.UserHomeDir()
```

Comment:
Perhaps you need to replace all `user.Current()` references in this file with `os.UserHomeDir`